tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/core/portal/PortalServerBehavior.java

```diff
@@ -908,10 +962,36 @@ public void startModule( IModule[] modules, IProgressMonitor monitor ) throws Co
         startOrStopModules( modules, "start", monitor );
     }
 
-    @Override
-    public void stopModule( IModule[] modules, IProgressMonitor monitor ) throws CoreException
+    public void startModules()
```

Comment:
Because I  found we have this issue is the moudle status didn't changed after we stopped server, the module state always keep started. So we change this. Andy also found this issue too.